server/methods/accounts/accounts.js

```diff
@@ -291,7 +291,9 @@ export function validateAddress(address) {
     } else {
       // No address, fail validation
       validated = false;
-      validatedAddress.failedValidation = false;
+      validatedAddress = {
+        failedValidation: false
+      };
     }
   }
   const validationResults = { validated, fieldErrors: validationErrors, formErrors, validatedAddress };
```

Comment:
previously, this method returned the `address` passed to it, but with your change, it could be replaced with `{ failedValidation: false }`. This could be fine (I have no idea what the downstream code looks like), but feels like a potentially breaking change.
To be safe, consider checking that `validatedAddress` is null/undefined,
``` JavaScript
// line 294:
if (!validatedAddress) {
  validatedAddress = {};
}
validatedAddress.failedValidation = false; // same as it was
```

Reply:
oops! my bad. I thought this was the `else` for `if (validator)`